message_sender/tests.py

```diff
@@ -2753,3 +2754,85 @@ def test_update_identity_by_sql(self):
         self.prepare_data()
         call_command('update_identity_field', '--loop', 'SQL')
         self.check_data()
+
+
+class TestAggregateOutbounds(AuthenticatedAPITestCase):
+    def test_aggregate_outbounds(self):
+        """
+        The aggregate outbounds task should create new AggregateOutbounds
+        objects that represent the current Outbounds
+        """
+        c1 = Channel.objects.create(channel_id="c1", configuration={})
+        c2 = Channel.objects.create(channel_id="c2", configuration={})
+
+        o = self.make_outbound(channel=c1)
+        o = Outbound.objects.get(id=o)
+        o.created_at = datetime(2017, 1, 1)
+        o.attempts = 2
+        o.save()
+
+        o = self.make_outbound(channel=c1)
+        o = Outbound.objects.get(id=o)
+        o.created_at = datetime(2017, 1, 1)
+        o.save()
+
+        o = self.make_outbound(channel=c2)
+        o = Outbound.objects.get(id=o)
+        o.created_at = datetime(2017, 1, 1)
+        o.save()
+
+        o = self.make_outbound(channel=c2)
+        o = Outbound.objects.get(id=o)
+        o.created_at = datetime(2017, 1, 1)
+        o.delivered = True
+        o.save()
+
+        o = self.make_outbound(channel=c2)
+        o = Outbound.objects.get(id=o)
+        o.created_at = datetime(2017, 1, 3)
+        o.save()
+
+        tasks.aggregate_outbounds('2017-01-01', '2017-01-02')
+
+        agg1 = AggregateOutbounds.objects.get(
```

Comment:
Maybe just do a count on `AggregateOutbounds` here so we are sure `o.created_at = datetime(2017, 1, 3)` is not included.